.github/workflows/code_size_comment.yml

```diff
@@ -27,7 +28,10 @@ jobs:
             });
 
             if (matchArtifact.length === 0) {
-              return false;
+              console.log('no matching artifact found');
```

Comment:
I think the `return artifact.name == "code-size-report"` on line 26/27 needs to be changed as well.

Reply:
I'm not sure I understand the need for this. It matches the name from the upload at:

https://github.com/micropython/micropython/blob/6503cd47afae82e7839d1f8456a397c2835bdf0b/.github/workflows/code_size.yml#L37

And it has been tested as working.

Reply:
Sorry, that wasn't clear.  I mean shouldn't it return a string not a bool?

```js
if (artifact.name == "code-size-report") {
    return "ok";
} else {
    return "fail";
}
```

Reply:
This return statement is for a lambda function for filtering an array. It is not a return statement for the top level function.

Reply:
Right you are!  Obviously I'm not very good at Javascript :)